### PR TITLE
Assignment 4 @ Mmoran314

### PR DESCRIPTION
**Pull request by slanante1997**
Work in progress. Have not yet implemented the backend logic assigned by assignment 4.

Our site:
https://yellow-coast-01713141e.3.azurestaticapps.net/

### CODE REVIEW

**Review by Nolan-Posey — COMMENTED** [review]
Make all pages responsive ✅
1080p desktop ✅
iPad Air ✅
Samsung Galaxy S20 Ultra Super Mega Excellent ✅
IPhone SE ✅
Create a improved landing page and site theme (Background, make it look attractive to play) ✅
Theme should extend all pages ✅
Add an instructions page that is available via the hamburger menu that details the features of the game and how to play ✅
Create a new Wordle game page that reuses components but will provide the same word for a specific day. Everyone going to the page that day will get the same word. The page should indicate Wordle of the Day or something like that ✅
Create a page that lists the last 10 daily words with statistics. Number of plays, Average score, Average time. (just list the date, don't show the word silly) ❌
It should indicate if the user has played that day ❌
Clicking on the item should take the player to that game (Extra Credit) ❌

File: Wordle.Api/Wordle.Api.Tests/PlayerServiceTests.cs [hunk 1]
```diff
@@ -5,40 +5,73 @@
 using System.Text;
 using System.Threading.Tasks;
 using Wordle.Api.Data;
+using Wordle.Api.Dtos;
 using Wordle.Api.Services;
 
-namespace Wordle.Api.Tests
+namespace Wordle.Api.Tests;
```
Comment (mmwoodfo) [inline Wordle.Api/Wordle.Api.Tests/PlayerServiceTests.cs:11]: You have a bunch of formatting changes - this make your git diff difficult to read. Make sure the changes you make to your assignment are relevant to the assignment.

File: wordle-web/src/App.vue [hunk 2]
```diff
@@ -19,26 +19,43 @@
...
+                <RouterLink :to="{ name: 'about' }"> About </RouterLink>
+              </v-list-item-title>
             </v-list-item>
           </v-list>
         </v-menu>
       </template>
     </v-app-bar>
 
-    <v-main>
+    <v-main style="v-main">
```
Comment (mmwoodfo) [inline wordle-web/src/App.vue:58]:
```suggestion
    <v-main>
```

**Review by mmwoodfo — COMMENTED** [review]
image

Getting some errors while calling your API

**Pull request closed without merging**